fusequery/query/src/datasources/remote/remote_database.rs

```diff
@@ -53,7 +53,10 @@ impl Database for RemoteDatabase {
     fn get_table(&self, _table_name: &str) -> Result<Arc<TableMeta>> {
         match self.tables.read().name2meta.get(_table_name) {
             Some(tbl) => Ok(tbl.clone()),
-            None => Err(ErrorCode::UnknownTable(_table_name)),
+            None => Err(ErrorCode::UnknownTable(format!(
+                "Unknown table: '{}'",
+                _table_name
```

Comment:
maybe `_table_name -> table_name` is better

Reply:
done